Formula/bareos-fd.rb

```diff
@@ -0,0 +1,72 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/17.2.4.tar.gz"
+  sha256 "4c443539012cf5ebb0fdb18878e604e82b951e6429c618acd18762f3c5724799"
+
+  depends_on "readline"
+  depends_on "openssl"
+
+  def install
+    system "./configure",
+    "--prefix=#{prefix}",
+    "--sbindir=#{bin}",
+    "--with-working-dir=#{var}/lib/bareos",
+    "--with-archivedir=#{var}/bareos",
+    "--with-confdir=#{etc}/bareos",
+    "--with-configtemplatedir=#{lib}/bareos/defaultconfigs",
+    "--with-scriptdir=#{lib}/bareos/scripts",
+    "--with-plugindir=#{lib}/bareos/plugins",
+    "--with-fd-password=XXX_REPLACE_WITH_CLIENT_PASSWORD_XXX",
+    "--with-mon-fd-password=XXX_REPLACE_WITH_CLIENT_MONITOR_PASSWORD_XXX",
+    "--with-basename=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
+    "--with-hostname=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
+    "--with-python",
+    "--enable-client-only"
+
+    system "make"
+    # * Makefile is intended to create a .pkg file.
+    #   As it is not required or used it's removed.
+    rm "platforms/osx/Makefile"
```

Comment:
why do we need to remove it? If it's not used, it's not used.

Reply:
The .pkg is not used. This makefile is intended to build bareos for mac os in other build environments. As we can ensure that this is an os x system and it would possibly interfere with the homebrew building process we remove it.

Reply:
Have you actually seen it interfere in some way or is that speculation about something that could happen?

Reply:
yes, a "make install" will break the build process (at least creates unwanted files).

Reply:
A separate make target would be far preferable to having to delete this file.

Reply:
Fixing `platforms/osx/Makefile` would be the best option. However, with the current release we are sticked to this. If it helps, I can create a bug report at https://bugs.bareos.org requesting this change and reference it here in the comment. As soon as this have been fixed in upstream, we can get rid of this rm command.

Reply:
:+1:

Reply:
Okay, I created: https://bugs.bareos.org/view.php?id=900
I leave it to @thraizz to integrate this into this pull request.